contracts/interfaces/ISTETH.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.2;
+
+interface IWSTETH {
+    function getStETHByWstETH(uint256 _amount) external view returns (uint256);
```

Comment:
please use for reference for getStETHByWstETH, getWstETHByStETH, etc.
wsteth: https://etherscan.io/address/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#code
steth: https://etherscan.io/address/0x20dc62d5904633cc6a5e34bec87a048e80c92e97#code